docs/server/auth.md

```diff
@@ -212,7 +212,7 @@ permissions.
 
 ### Try to access your table using DuckDB
 
-First, let’s give our user account (`bobbie@rocinante`) `USE CATALOG`, `USE` `SCHEMA`, and `SELECT` permissions.
```

Comment:
nit: This is auto-generated? not sure why something decided to put a \` \` between `USE` and `SCHEMA`?